### PR TITLE
IGVF-1062-matrix-file-facets

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/searches/configs/matrix_file.py [hunk 1]
```diff
@@ -6,6 +6,67 @@
 )
 def matrix_file():
     return {
+        'facets': {
+            'uuid': {
+                'title': 'UUID'
```
Comment (jenjou) [inline src/igvfd/searches/configs/matrix_file.py:11]: UUID and accession don't seem to be included in facets for other File subclasses except Signal File. I'm confused if we should be adding UUID and accession to every subclass, or if we should be omitting them?

File: src/igvfd/searches/configs/matrix_file.py [hunk 2]
```diff
@@ -25,6 +86,12 @@ def matrix_file():
             'lab': {
                 'title': 'Lab'
             },
+            'file_set': {
+                'title': 'File Set'
+            },
+            'content_summary': {
+                'title': 'Content Summary'
+            },
             'status': {
```
Comment (jenjou) [inline src/igvfd/searches/configs/matrix_file.py:95]: Should upload_status also be added to columns?

File: src/igvfd/searches/configs/matrix_file.py (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/searches/configs/matrix_file.py:11]: Yes I coped these from signal file, thanks for catching that I don't see the use in having uuid and accession in the facets. It looks like Mingjie had added them there but I don't think they serve any purpose on the search page so I will remove them.

File: src/igvfd/searches/configs/matrix_file.py (hunk 2 above)
Comment (ian-whaling) [reply src/igvfd/searches/configs/matrix_file.py:95]: upload_status is already in the columns

**Review by jenjou — APPROVED** [review]

**Pull request merged**